RCSwitch.h

```diff
@@ -125,6 +125,7 @@ class RCSwitch {
         HighLow syncFactor;
         HighLow zero;
         HighLow one;
+        boolean bLowFirst;
```

Comment:
Please don't use hungarian notation, it clearly does not match the surrounding code.